meerk40t/balormk/controller.py

```diff
@@ -339,6 +340,7 @@ def connect_if_needed(self):
                     raise ConnectionError
                 self.init_laser()
             except (ConnectionError, ConnectionRefusedError):
+                time.sleep(0.3)
```

Comment:
We hammer the connect less hard with a tiny sleep each time we failed to connect.